tethys_cli/start_commands.py

```diff
@@ -87,48 +82,6 @@ def quickstart_command(args):
     if not Path(db_config_options["db_name"]).exists():
         configure_tethys_db(**db_config_options)
 
-    setup_django()
```

Comment:
Just to verify: this is removed now bc users can scaffold in the portal after it starts, yes?

Reply:
That was removed as part of removing the "Hello World" app being auto-scaffolded-and-installed with `tethys quickstart`. I feel that is no longer necessary with the GUI scaffold. The documentation has been updated as such.